### DISCUSSION TITLE
السؤال عن ترجمة backend

### DISCUSSION THREAD

**Discussion by atefBB** [softvenue/i18n#78] [Q&A]
<div dir="rtl">هل من اقتراح لترجمة مصطلح backend ؟</div>
@softvenue/all

**Comment by celyes** [comment]
هناك عدة مصطلحات. سأوافيك بالانسب منها و يمكننا الاتفاق على أحدها
@softvenue/all

**Comment by atefBB** [comment]
<div dir="rtl">وجدت مصطلح "منتهى خلفي" أو "نهاية خلفية" في قاموس عربآيز، لكنني حقيقة أحسه ترجمة حرفية للكلمة.</div>

**Comment by celyes** [comment]
سأقوم بوضع مصطلح له اذا استطعت
